ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -8,36 +8,14 @@
 #include <string>
 #include "frontend.hpp"
 #include "frontend_manager_defs.hpp"
+#include "ngraph/variant.hpp"
 
 namespace ngraph
 {
     namespace frontend
     {
-        /// Capabilities for requested FrontEnd
-        /// In general, frontend implementation may be divided into several libraries by capability
-        /// level It will allow faster load of frontend when only limited usage is expected by
-        /// client application as well as binary size can be minimized by removing not needed parts
-        /// from application's package
-        namespace FrontEndCapabilities
-        {
-            /// \brief Just reading and conversion, w/o any modifications; intended to be used in
-            /// Reader
-            static const int FEC_DEFAULT = 0;
-
-            /// \brief Topology cutting capability
-            static const int FEC_CUT = 1;
-
-            /// \brief Query entities by names, renaming and adding new names for operations and
-            /// tensors
-            static const int FEC_NAMES = 2;
-
-            /// \brief Partial model conversion and decoding capability
-            static const int FEC_WILDCARDS = 4;
-        }; // namespace FrontEndCapabilities
-
         // -------------- FrontEndManager -----------------
-        using FrontEndCapFlags = int;
-        using FrontEndFactory = std::function<FrontEnd::Ptr(FrontEndCapFlags fec)>;
+        using FrontEndFactory = std::function<FrontEnd::Ptr()>;
```

Comment:
In this case probably 'factory' method is not needed at all, and FrontEndManager can simply store list of created frontends, as FrontEnd is not heavy and can be created during initializatoin. But maybe we can do it by separate PR

Reply:
Lets do it in different PR